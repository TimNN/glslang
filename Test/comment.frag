	 
	  //
/* anotehn t* ontuh * / tnoahnt /* oo */
/* multi line...

ao */
/*  no escape \
oanot */
// escape nothing \o oeu
// escape newline \
still in a comment

