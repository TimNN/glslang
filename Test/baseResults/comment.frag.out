0:? Sequence
0:17  Function Definition: main( (void)
0:17    Function Parameters: 
0:?   Linker Objects
0:?     'v' (smooth in 4-component vector of float)

